lib/fileio/jobs/readJob.cpp

```diff
@@ -60,14 +60,12 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 UsdMaya_ReadJob::UsdMaya_ReadJob(
         const std::string &iFileName,
-        const std::string &iPrimPath,
-        const std::map<std::string, std::string>& iVariants,
+        const MayaUsd::ImportData &iImportData,
```

Comment:
You'll see a few changes like this. I've replaced the primPath and variants arg, with my new import data (which contains both those and more).